common/sgx/verifier.c

```diff
@@ -185,7 +185,7 @@ static oe_result_t _process_sgx_report_data(
     return result;
 }
 
-static oe_result_t _add_claim(
+oe_result_t oe_sgx_add_claim(
```

Comment:
Need to check the naming convention. Maybe it should be "sgx_attestation_plugin_add_Claim"